src/SFA.DAS.ApprenticeCommitments.Web.UnitTests/ApprenticeIdFromRegisatrionIdTests.cs

```diff
@@ -1,11 +1,11 @@
-﻿using FluentAssertions;
+﻿using System;
+using System.Security.Claims;
+using FluentAssertions;
 using NUnit.Framework;
-using SAF.DAS.ApprenticeCommitments.Web.UnitTests.AutoFixtureCustomisations;
 using SFA.DAS.ApprenticeCommitments.Web.Services;
-using System;
-using System.Security.Claims;
+using SFA.DAS.ApprenticeCommitments.Web.UnitTests.AutoFixtureCustomisations;
 
-namespace SAF.DAS.ApprenticeCommitments.Web.UnitTests
+namespace SFA.DAS.ApprenticeCommitments.Web.UnitTests
```

Comment:
I wish we wouldn't make large-scale renaming changes like this as part of other work.  Ideally they would be in their own PR so as not to obfuscate the actual story related changes.